File: 2.0/python/calls/getCallRecordingByName.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.calls.getCallRecordingByName(
    id=379506003,
    name='call-recording1',
    # return only specific fields
    fields='callId,campaignId,lengthInBytes,mp3Url'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/calls/getCallRecordingByName.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getCallRecordingByName();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003,
                                                                "name" => "call-recording1"));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "callId,campaignId,lengthInBytes,mp3Url"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
